tldextract/tldextract.py

```diff
@@ -39,6 +38,7 @@
 import socket
 import warnings
 
+import idna.codec
```

Comment:
because it is a third party import.  this is the relevant pep8:

https://www.python.org/dev/peps/pep-0008/#imports

Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.

no complaints from it.  consistent with the packages documented use (https://pypi.python.org/pypi/idna) and i'm currently using my branch and it works just fine.